lang/en.yml

```diff
@@ -163,10 +163,10 @@ en:
     REDIRECTTOPAGE: 'A page on your website'
     SINGULARNAME: 'Redirector Page'
     YOURPAGE: 'Page on your website'
-    db_ExternalURL: 'External URL'
-    db_RedirectionType: 'Redirection type'
-    has_one_LinkTo: 'Link to'
-    has_one_LinkToFile: 'Link to file'
+    db_ExternalURL: 'Other website URL'
```

Comment:
We shouldn't change translation source values, as when we tx push it'll delete any existing translations

Create new keys if you need to change values

Reply:
See https://github.com/silverstripe/silverstripe-cms/pull/2983#discussion_r1713062292

I've linked to that comment from other PRs that you've approved, so presumably you broadly agree with it?
Note that `db` keys and `has_one` keys are directly used in `fieldLabels()` in a dynamic way. We can't really substitute them with other keys in a particularly clean way or I'd just use the existing ones that I'm intentionally swapping away from (again see the comment linked above).